src/azure-cli/azure/cli/command_modules/iot/tests/hybrid_2020_09_01/test_iot_commands.py

```diff
@@ -0,0 +1,550 @@
+
+        # Test 'az iot hub routing-endpoint delete'
+        self.cmd('iot hub routing-endpoint delete --hub-name {0} -g {1} -n {2}'.format(hub, rg, endpoint_name),
+                 checks=[self.check('length(eventHubs[*])', 0),
+                         self.check('length(serviceBusQueues[*])', 0),
+                         self.check('length(serviceBusTopics[*])', 0),
+                         self.check('length(storageContainers[*])', 1)])
+
+        # Test 'az iot hub devicestream show'
+        self.cmd('iot hub devicestream show -n {0} -g {1}'.format(hub, rg), checks=self.is_empty())
+
+        # Test 'az iot hub message-enrichment create'
+        real_endpoints = 'events'
+        fake_endpoints = 'events fake_endpoint'
+        key = 'key'
+        fake_key = 'fake_key'
+        value = 'value'
+
+        self.cmd('iot hub message-enrichment create -n {0} -g {1} --key {2} --value {3} --endpoints {4}'
+                 .format(hub, rg, key, value, fake_endpoints), expect_failure=True)
+        self.cmd('iot hub message-enrichment create -n {0} -g {1} --key {2} --value {3} --endpoints {4}'
+                 .format(hub, rg, key, value, real_endpoints), checks=[self.check('length(properties.routing.enrichments)', 1)])
+
+        # Test 'az iot hub message-enrichment update'
+        self.cmd('iot hub message-enrichment update -n {0} -g {1} --key {2} --value {3} --endpoints {4}'
+                 .format(hub, rg, fake_key, value, real_endpoints), expect_failure=True)
+        self.cmd('iot hub message-enrichment update -n {0} -g {1} --key {2} --value {3} --endpoints {4}'
+                 .format(hub, rg, key, value, fake_endpoints), expect_failure=True)
+        self.cmd('iot hub message-enrichment update -n {0} -g {1} --key {2} --value {3} --endpoints {4}'
+                 .format(hub, rg, key, value, real_endpoints), checks=[self.check('length(properties.routing.enrichments)', 1)])
+
+        # Test 'az iot hub message-enrichment list'
+        self.cmd('iot hub message-enrichment list -n {0} -g {1}'.format(hub, rg),
+                 checks=[self.check('length([*])', 1)])
+
+        # Test 'az iot hub message-enrichment delete'
+        self.cmd('iot hub message-enrichment delete -n {0} -g {1} --key {2}'.format(hub, rg, fake_key),
+                 expect_failure=True)
+        self.cmd('iot hub message-enrichment delete -n {0} -g {1} --key {2}'.format(hub, rg, key),
+                 checks=[self.check('length(properties.routing.enrichments)', 0)])
+
+        # Test 'az iot hub manual-failover'
+        self.cmd('iot hub manual-failover -n {0} -g {1}'.format(hub, rg),
+                 checks=[self.check('location', location)])
+        # Test 'az iot hub delete'
+        self.cmd('iot hub delete -n {0}'.format(hub), checks=self.is_empty())
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(location='westus2')
+    @StorageAccountPreparer()
+    @unittest.skip('Need to try with passing SAS key in environment.')  # TODO
```

Comment:
Skipped tests.

Reply:
iot test was taken from master, few of them is not supported in 2020-09-01-hybrid and hence it was marked for skipped.
i think the skip description is not very accurate. i will remove these test cases in next PR.